### ISSUE TITLE
feature: mobile friendly layout

### ISSUE THREAD

**Issue by blurymind** [piskelapp/piskel#1080]
Bug description

(this template is intended for bug reports, erase it when requesting features/enhancements)

### Steps to reproduce the bug
1. go to piskelapp.com
2. open dev tools and switch on mobile preview
3. Piskel is unusable for smartphones

### Environment details
* operating system: any
* browser (or offline application version): chromium

### Sprite details
* number of frames: na
* sprite resolution: na
* session duration: na

Feel free to include a screenshot of the bug, a .piskel file of the sprite or anything else that can help us investigate.

![image](https://user-images.githubusercontent.com/6495061/204899301-d056df8a-7749-4505-8dc0-4d3ace044770.png)

It would be nice as a first step to make the UI more responsive to vertical mode smartphones, then think about possible touch input improvements.

Basically if piskel is mobile friendly, it would become worthy of being used as a progressive web app on smartphones
